corehq/apps/receiverwrapper/tests/test_auth.py

```diff
@@ -323,3 +330,53 @@ def setUp(self):
     def tearDown(self):
         self.user.delete(self.domain, deleted_by=None)
         super(InsecureAuthTest, self).tearDown()
+
+
+def return_200(*args, **kwargs):
+    return HttpResponse("success!", status=200)
+
+
+@mock.patch('corehq.apps.receiverwrapper.views._process_form', new=return_200)
+class TestAPIEndpoint(TestCase):
+    domain = "submission-api-test"
+    username = 'samiam'
+    password = 'p@$$w0rd'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.domain_obj = create_domain(cls.domain)
+        cls.url = reverse(post_api, args=[cls.domain])
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
+
+    def _create_user(self, edit_data=False, access_api=False):
+        role = UserRole.create(
+            self.domain, 'api-user', permissions=Permissions(
+                edit_data=edit_data, access_api=access_api
+            )
+        )
+        web_user = WebUser.create(self.domain, self.username, self.password,
+                                  None, None, role_id=role.get_id)
+        self.addCleanup(lambda: web_user.delete(None, None))
```

Comment:
You don't need the lambda:
```suggestion
        self.addCleanup(web_user.delete, None, None)
```

Reply:
oh neat, I didn't know that took args